app/services/report_errors_to_argo.rb

```diff
@@ -0,0 +1,24 @@
+class ReportErrorsToArgo
```

Comment:
I'm inclined to substitute 'workflow' for 'argo' as it's perhaps more accurate and more descriptive.  But wonder if that makes sense when we get around to adding the code to do the solr indexing ... again - perhaps debate pros and cons in person.  `services/dor_workflows.rb` ??

Reply:
i think i agree with this... ultimately, the errors are being consumed by argo from WFS, but as far as pres catalog knows, it's just updating some workflow info (which, theoretically, could be consumed by things other than argo, even if that's the only service that cares to consume them for now).